UnityProject/Assets/Scripts/Player/Mind.cs

```diff
@@ -148,6 +148,14 @@ public void SetAntag(SpawnedAntag newAntag)
 		body.OrNull()?.GetComponent<PlayerOnlySyncValues>().OrNull()?.ServerSetAntag(true);
 	}
 
+	public void AddObjectiveToAntag(Objective objectiveToAdd)
```

Comment:
this still needed?

Reply:
Might have some use in the future because we currently don't have a way to add specific objectives to antags from admins or coders.

Reply:
only thing I would say would maybe add some code for notifying the player of added objectives, you can just add a "TODO needs to notify players/client"

Reply:
done